dcor/homogeneity.py

```diff
@@ -164,7 +168,13 @@ def statistic_function(distance_matrix):
 
 def energy_test(*args, **kwargs):
     """
-    energy_test(*args, num_resamples=0, exponent=1, random_state=None)
+    energy_test(
```

Comment:
You need to escape the newline in order for Sphinx to know that this line is the prototype of the function. Check `distance_covariance_test` to see how to do that. You can generate the docs locally using `make html` in the docs folder, to check that the documentation for this function is done properly.